content/webapp/services/prismic/transformers/card.ts

```diff
@@ -0,0 +1,23 @@
+import { CardPrismicDocument } from '../types/card';
+import {
+  parseTitle,
+  parseFormat,
+  asText,
+  checkAndParseImage,
+  parseLink,
+} from '@weco/common/services/prismic/parsers';
+import { Card } from '@weco/common/model/card';
+
+export function transformCard(document: CardPrismicDocument): Card {
+  const { title, format, description, image, link } = document.data;
+
+  return {
+    type: 'card',
+    title: parseTitle(title),
+    format: parseFormat(format),
+    description: asText(description),
+    image: image ? checkAndParseImage(image) : null,
+    link: parseLink(link),
+    order: undefined,
```

Comment:
do you have to specify that this is `undefined`, or can you just leave it off altogether?